group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupConfig.java

```diff
@@ -41,10 +41,14 @@ public class GroupConfig extends AbstractConfig {
 
     public static final String CONSUMER_HEARTBEAT_INTERVAL_MS_CONFIG = "consumer.heartbeat.interval.ms";
 
+    public static final String SHARE_RECORD_LOCK_DURATION_MS = "share.record.lock.duration.ms";
+
     public final int consumerSessionTimeoutMs;
 
     public final int consumerHeartbeatIntervalMs;
 
+    public final int shareRecordLockDurationMs;
+
     private static final ConfigDef CONFIG = new ConfigDef()
```

Comment:
The way the KIP is written, if the group config is not set, it takes the default value from the broker config. I don't think this quite says that. It says that the group config's default is the broker config's default, so if I modified the broker config with the intention of affecting all share groups, it would not work.